README.md

```diff
@@ -18,24 +22,24 @@ defp deps do
 end
 ```
 
-From PropcCheck 1.3.0 onwards, we require at least Elixir 1.7 since in Elixir 1.11 function `get_stracktrace()` is deprecated. However, we 
+From PropcCheck 1.3.0 onwards, we require at least Elixir 1.7 since in Elixir 1.11 function `get_stracktrace()` is deprecated. However, we
 only support OTP 22, since `:proper` does not yet support OTP 23 in a released version. Hopefully, this will change soon! Running under
-OTP 23 results in no stacktraces - which is not helpful for identifying bugs. 
+OTP 23 results in no stacktraces - which is not helpful for identifying bugs.
 
 ## Changes
 
-Relevant changes are documented in the [Changelog](changelog.html), on [GitHub
+Relevant changes are documented in the [Changelog](https://hexdocs.pm/propcheck/changelog.html), on [GitHub
```

Comment:
In which scenario does this not work? I added both links to provide proper links for github, Dash and hexdocs - at least this was the plan

Reply:
In the current https://github.com/alfert/propcheck/blob/master/README.md, the `Changelog` link shows 404, which points to https://github.com/alfert/propcheck/blob/master/changelog.html